graphql/schema.graphql

```diff
@@ -172,5 +173,24 @@ type User {
   displayName: String!
 }
 
+type RecentTaskLogs {
```

Comment:
Are you planning on adding more to this in the future? I'm wondering if we don't need this type, and can just return the array

Reply:
I plan on adding agentLogs, systemLogs and taskLogs